### PR TITLE
Added SID to endpoints

### PR DESCRIPTION
**Pull request by ShirMon**
Added SID dir creation and download location via the multer middleware,
& edited API endpoints to include SID

### CODE REVIEW

File: src/application.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import multer from 'multer';
 import path from 'path';
 import {FILE_UPLOAD_SERVICE, STORAGE_DIRECTORY} from './keys';
 import {MySequence} from './sequence';
+const fs = require('fs')
```
Comment (itamar-marom) [inline src/application.ts]: why const ? if you're on ts isn't it supoose to be:
import fs from 'fs' ?

**Review by itamar-marom — COMMENTED** [review]
I left you a side note. But it's not that important

File: src/application.ts (hunk 1 above)
Comment (itamar-marom) [reply src/application.ts]: @ShirMon

Comment (ShirMon) [reply src/application.ts]: Done, it doesn't make much a difference but the ts standard is import from so there you go.

**Review by itamar-marom — APPROVED** [review]

**Pull request merged**